sqlglot/dialects/mysql.py

```diff
@@ -867,3 +867,29 @@ def chr_sql(self, expression: exp.Chr) -> str:
             charset = expression.args.get("charset")
             using = f" USING {self.sql(charset)}" if charset else ""
             return f"CHAR({this}{using})"
+
+        def timestamptrunc_sql(self, expression: exp.TimestampTrunc) -> str:
+            this = self.sql(expression, "this")
+            unit = expression.args.get("unit")
+            unix_timestart = "'1900-01-01 00:00:00'"
```

Comment:
```suggestion
            unix_timestart = exp.Literal.string('1900-01-01 00:00:00')
```

Reply:
We should comment that this value was arbitrarily picked as an "old-enough date" (AFAICT), so as to avoid getting back a negative number from the `TIMESTAMPDIFF`.

Let's also rename this to something better while we're at it - `unix_timestart` is a little confusing given that this isn't 1970-01-01. I'd probably do `start_ts` & add this comment to clear any confusion.